web/src/plugins/pipelines/useDnD.js

```
@@ -0,0 +1,510 @@
+  
+  
+ 
+  const filteredOptions = ref(formattedOptions.value);
+
+const filterOptions = (val, update) => {
+  if (val === '') {
+    filteredOptions.value = formattedOptions.value;
+  } else {
+    const filterFn = option =>
+      option.label.toLowerCase().includes(val.toLowerCase()) ||
+      (option.node_type && option.node_type.toLowerCase().includes(val.toLowerCase()));
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)